buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -46,19 +46,20 @@
  */
 public abstract class AbstractByteBuf extends ByteBuf {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(AbstractByteBuf.class);
-    private static final String LEGACY_PROP_CHECK_ACCESSIBLE = "io.netty.buffer.bytebuf.checkAccessible";
-    private static final String PROP_CHECK_ACCESSIBLE = "io.netty.buffer.checkAccessible";
+    private static final String LEGACY_PROP_CHECK_ACCESSIBLE = "siren.io.netty.buffer.bytebuf.checkAccessible";
+    private static final String PROP_CHECK_ACCESSIBLE = "siren.io.netty.buffer.checkAccessible";
     static final boolean checkAccessible; // accessed from CompositeByteBuf
-    private static final String PROP_CHECK_BOUNDS = "io.netty.buffer.checkBounds";
+    private static final String PROP_CHECK_BOUNDS = "siren.io.netty.buffer.checkBounds";
     private static final boolean checkBounds;
 
     static {
+        // siren: set the default value to false
```

Comment:
Maybe add mention to why we need those values to be disabled

Reply:
I think we can put that in `README.md`: there is a section at the start `Changes` that lists what modifications the netty fork brings. For this particular comment, the default value of `check_accessible` is changed to `false` because we don't want any runtime checks for performance reason.

Reply:
Done here https://github.com/sirensolutions/netty/pull/29/commits/80abb4e6388788f0599acd0787d7d542b179ae2c